cookbooks/aws-parallelcluster-environment/files/amazon-2023/network_interfaces/configure_nw_interface.sh

```diff
@@ -24,8 +24,8 @@ fi
 
 cd "$configuration_directory"
 
-SUFFIX=$(printf "%03d" $NETWORK_CARD_INDEX)$(printf "%03d" $DEVICE_NUMBER)
-ROUTE_TABLE=1${SUFFIX}
+SUFFIX=$(printf "%03d" $NETWORK_CARD_INDEX)$(printf "%02d" $DEVICE_NUMBER)
+ROUTE_TABLE="$(( $SUFFIX + 10 ))"
```

Comment:
it would be good to have a comment in the code to describe why this is required.